README.md

```diff
@@ -120,17 +120,17 @@ Integration testing requires a local MongoDB and Atlas Data Lake instance to be
 ```
 
 ### Running Local mongod and Atlas Data Lake
```

Comment:
```suggestion
### Running Local mongod and Atlas Data Federation
```

Reply:
Good catch -- I was so focused on the acronym, I forgot to look for the actual words 😅 I'll make this change and do an audit myself to make sure I didn't miss any others

Reply:
updated